build.sh

```diff
@@ -112,6 +112,8 @@ build-all-couch() {
       CONTAINERARCH="${arch}" build-couch ${base}
     done
   done
+  CONTAINERARCH="s390x" build-couch centos-8
+  CONTAINERARCH="s390x" build-couch centos-9
```

Comment:
Would it work to add centos-8 and centos-9 target to `XPLAT_BASES="debian-bullseye ubuntu-focal"` so it would be `XPLAT_BASES="debian-bullseye ubuntu-focal centos-8 centos-9"`

Reply:
@nickva Yes, I think we can also add ubuntu-jammy target here since the multi-arch `apache/couchdbci-ubuntu:jammy-erlang-24.3.4.10` image is available now.

I'll test on the local Intel VM to see if it works.

Reply:
Oh great, idea @kun-lu20, we can add ubuntu-jammy too now. Thanks for mentioning it!

Reply:
Hi @nickva ,

I've added `ubuntu-jammy centos-8 centos-9` target to `XPLAT_BASES` and tested to generate all packages via running command `./build.sh couch-all https://dist.apache.org/repos/dist/release/couchdb/source/3.3.2/apache-couchdb-3.3.2.tar.gz` on an Intel VM.

All packages across various arches were successfully built, except RHEL8 rpm package on arm64, because SpiderMonkey 60 is known broken on ARM 64, so this package is skipped in `build-all-couch()`.

I've verified the generated RHEL 8/9 RPM packages and Ubuntu-jammy package on Intel and s390x VMs, following the instructions in https://docs.couchdb.org/en/stable/intro/tour.html. All packages worked fine.

Please take a look at the code changes when you have some time. Thank you very much!